firefish/fluids.py

```diff
@@ -0,0 +1,54 @@
+"""
+This module provides shortcuts to commonly used fluids
+"""
+
+import enum
+
+from firefish.case import (
+    FileName)
+
+class Fluid(enum.Enum):
+    """
+    An enumeration of commonly used fluids
+
+    AIR generates the recommended OpenFoam thermophysicalProperties for air.
+
+    DIMENSIONLESS_AIR generates a normalised gas whith gamma=7/5 and with
+    the property that at 1 temperature unit the speed of sound is 1
+    velocity unit"""
+    AIR = 0
+    DIMENSIONLESS_AIR = 1
+
+def write_thermophysical_properties(case, fluid):
+    """"
+    Writes a thermophysicalProperties dict in the given case for the
+    specified fluid.
+
+    Args:
+        case (firefish.case.Case): the case in which to write the dict
+        fluid (firefish.fluids.Fluid): the fluid to use
+    """
+
+    thermo_dict = None
+
```

Comment:
Can we get a comment with the source for these values? Probably a link to the OpenFOAM docs?